### PR TITLE
IBX-4760: Unsuported ezstarrating in upgrade

### PR DESCRIPTION
**Pull request by DominikaK**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://issues.ibexa.co/browse/IBX-4760
| Versions      | all
| Edition       | all

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md [hunk 1]
```diff
@@ -242,6 +242,70 @@ Import to your database the changes provided in one of the following files. It's
 
 Steps here should only be done once you are ready to move away from legacy and Legacy Bridge, as the following Field Types are not supported by legacy. In other words, content you have migrated will not be editable in legacy admin interface anymore, but rather in the more modern eZ Platform back-end UI, allowing you to take full advantage of what the Platform has to offer.
 
+The Field Types unsupported in eZ Platform are:
+
+- [XmlText](#321-migrate-xmltext-content-to-richtext)
+- [Page](#322-migrate-page-field-to-page-ez-enterprise-only)
+- Star Rating
+- Matrix
```
Comment (vidarl) [inline docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md:250]: The matrix fieldtype is still supported as mentioned in "old" doc : https://github.com/ezsystems/developer-documentation/pull/1335/files#diff-c5a23f68142e5e2271c509d89f7a5e9d82d06d6c0fdf629b8fa509613e2bf0b2

However, in order to make it run on later 2.5 versions you need to run a migration script : https://doc.ibexa.co/en/latest/update_and_migration/from_1.x_2.x/update_db_to_2.5/#changes-to-matrix-field-type

**Review by vidarl — CHANGES_REQUESTED** [review]

File: docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md [hunk 2]
```diff
@@ -242,6 +242,69 @@ Import to your database the changes provided in one of the following files. It's
...
+!!! tip
+
+    To find out which Content Types use a specific Field Type,
+    you can run the following SQL query on your database (in this case, for the Star Rating Field Type):
+
+    ``` sql
+    select contentclass_id from ezcontentclass_attribute where data_type_string='ezsrrating' group by contentclass_id;
+    ```
+
+??? tip "`defaultLayout` setting not available"
```
Comment (vidarl) [inline docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md:265]: About this section.. This section applies if you upgrade to 2.5 and then realizes that you still have old page fields that hasn't been converted and you want to delete them. IE: you want to delete the fields using the admin-ui in 2.5. This also applies if you are intending to run 2.5 with legacy bridge. Then the config below should to be adapted to the config you have in legacy ( so adding any custom layouts  etc. ).

If you realize you have page fields you want to delete prior to migration, you may just do that in the legacy admin without applying the config below.
Sorry that I was not clear in this in our earlier chat

If you

File: docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md [hunk 3]
```diff
@@ -242,6 +242,69 @@ Import to your database the changes provided in one of the following files. It's
 
 Steps here should only be done once you are ready to move away from legacy and Legacy Bridge, as the following Field Types are not supported by legacy. In other words, content you have migrated will not be editable in legacy admin interface anymore, but rather in the more modern eZ Platform back-end UI, allowing you to take full advantage of what the Platform has to offer.
 
+The Field Types unsupported in eZ Platform are:
+
+- [XmlText](#321-migrate-xmltext-content-to-richtext)
+- [Page](#322-migrate-page-field-to-page-ez-enterprise-only)
+- Star Rating
+
+For Field Types which do not have specific procedures below, you must either:
```
Comment (dabrt) [inline docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md:251]: If you go with:
```suggestion
For Field Types which do not have specific procedures below, you must take one of the following courses of action:
```
you can remove the ", or" part and fullstop from the bulleted list below.

File: docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md [hunk 4]
```diff
@@ -242,6 +242,69 @@ Import to your database the changes provided in one of the following files. It's
...
+    To find out which Content Types use a specific Field Type,
+    you can run the following SQL query on your database (in this case, for the Star Rating Field Type):
+
+    ``` sql
+    select contentclass_id from ezcontentclass_attribute where data_type_string='ezsrrating' group by contentclass_id;
+    ```
+
+??? tip "`defaultLayout` setting not available"
+
+    If you want to remove a Page Field from your Content Type definition, you can encounter an issue
```
Comment (dabrt) [inline docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md:267]: Is it rather "When removing..."?

File: docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md [hunk 5]
```diff
@@ -242,6 +242,69 @@ Import to your database the changes provided in one of the following files. It's
...
+                            template: 3zoneslayout1.tpl
+                        3ZonesLayout2:
+                            name: 3 zones (layout 2)
+                            template: 3zoneslayout2.tpl
+                        CallForActionLayout:
+                            name: Call For Action zone layout
+                            template: callforactionlayout.tpl
+    ```
+
+    Clear and refresh the page, which should activate the dropdown.
```
Comment (dabrt) [inline docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md:301]:
```suggestion
    Clear the cache and refresh the page. The the dropdown should now be active.
```

**Review by dabrt — APPROVED** [review]
Please see comments

File: docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md (hunk 2 above)
Comment (DominikaK) [reply docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md:265]: Does this also refer to upgrade to 1.13? Or do you only do this on 1.13 > 2.5?

Comment (vidarl) [reply docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md:265]: It applies to the moment you drop legacy support ( ie move for ezpublish platform to platform, or get rid of legacy bridge in ezplatform )

File: docs/update_and_migration/from_1.x_2.x/update_db_to_2.5.md [hunk 6]
```diff
@@ -903,6 +903,50 @@ CREATE INDEX idx_workflow_name ON ezeditorialworkflow_workflows(workflow_name);
 
 [[% include 'snippets/update/notify_support.md' %]]
 
+??? tip "`defaultLayout` setting not available"
+
+    If you migrated you installation from eZ Publish Platform,
+    in Page Builder you can encounter an issue where the **Default layout** dropdown is disabled
```
Comment (vidarl) [inline docs/update_and_migration/from_1.x_2.x/update_db_to_2.5.md:909]: maybe better ? :
```suggestion
    you can in Page Builder encounter an issue where the **Default layout** dropdown is disabled
```

File: docs/update_and_migration/from_1.x_2.x/update_db_to_2.5.md [hunk 7]
```diff
@@ -903,6 +903,50 @@ CREATE INDEX idx_workflow_name ON ezeditorialworkflow_workflows(workflow_name);
...
+                            template: 3zoneslayout1.tpl
+                        3ZonesLayout2:
+                            name: 3 zones (layout 2)
+                            template: 3zoneslayout2.tpl
+                        CallForActionLayout:
+                            name: Call For Action zone layout
+                            template: callforactionlayout.tpl
+    ```
+
+    Clear the cache and refresh the page. The the dropdown should now be active.
```
Comment (vidarl) [inline docs/update_and_migration/from_1.x_2.x/update_db_to_2.5.md:943]:
```suggestion
    Clear the cache and refresh the page. The dropdown should now be active.
```

**Review by vidarl — CHANGES_REQUESTED** [review]

**Review by vidarl — APPROVED** [review]

**Pull request merged**